src/components/NavBar/ChainSelector.tsx

```diff
@@ -29,6 +30,10 @@ const NETWORK_SELECTOR_CHAINS = [
   SupportedChainId.BNB,
 ]
 
+if (!isProductionEnv()) {
+  NETWORK_SELECTOR_CHAINS.push(SupportedChainId.GOERLI)
+}
```

Comment:
if #6610 lands first we should swap this out for Sepolia

Reply:
just approved!